package.json

```diff
@@ -14,7 +14,7 @@
     "cypress:backend": "cd backend && yarn run dev",
     "cypress:webapp": "cd webapp && yarn run dev",
     "cypress:setup": "run-p cypress:backend cypress:webapp",
-    "cypress:run": "cross-env cypress run --browser chromium",
+    "cypress:run": "cross-env cypress run",
```

Comment:
Oh really? We can use electron again? We should remove the snap `chromium` from our build pipeline then.